containerd-shim-aspdotnet-v1/src/main.rs

```diff
@@ -111,7 +111,9 @@ pub fn prepare_module(
 }
 
 impl Instance for Wasi {
-    fn new(id: String, cfg: &InstanceConfig) -> Self {
+    type E = wasmtime::Engine;
+    fn new(id: String, cfg: Option<&InstanceConfig<Self::E>>) -> Self {
+        let cfg = cfg.unwrap();
```

Comment:
Is it correct to assume we don't have to deal w/ it being `None` in this use-case?